package.json

```diff
@@ -54,7 +54,6 @@
     "json-loader": "0.5.7",
     "mapbox-gl": "^1.11.1",
     "mapbox-gl-js-mock": "https://github.com/QwantResearch/mapbox-gl-js-mock.git#7eaee7b",
-    "masq-lib": "git+https://github.com/QwantResearch/masq-lib.git#v0.14.3",
```

Comment:
`"detect-browser"` can be removed too. 
It was only used in `src/libs/masq.js` to disable Masq on unsupported browsers.

Reply:
Thank you, it got past my radar indeed :)